src/core/lib/event_engine/posix_engine/timer_manager.h

```diff
@@ -44,30 +44,23 @@ namespace posix_engine {
 // all times, and thus effectively preventing the thundering herd problem.
 // TODO(ctiller): consider unifying this thread pool and the one in
 // thread_pool.{h,cc}.
-class TimerManager final : public grpc_event_engine::experimental::Forkable {
+class TimerManager final {
```

Comment:
Could you elaborate on why this is no longer a Forkable ?

Reply:
This class no longer needs to do anything special on fork, which is to say that it is trivially forkable.

Reply:
How is that true if it starts a thread?

Reply:
Ah right, the fact that it's non-joinable doesn't help the forking problem. Thanks both for the reminder.

Reply:
Fixed